static/js/vue-cdr-access/tests/unit/aggregateRecord.spec.js

```diff
@@ -0,0 +1,447 @@
+        fallbackLocale: 'en',
+        messages: translations
+    });
+
+    beforeEach(() => {
+        moxios.install();
+        router = createRouter({
+            history: createWebHistory(process.env.BASE_URL),
+            routes: [
+                {
+                    path: '/record/:uuid',
+                    name: 'displayRecords',
+                    component: displayWrapper
+                }
+            ]
+        });
+
+        wrapper = shallowMount(aggregateRecord, {
+            global: {
+                plugins: [i18n, router]
+            },
+            props: {
+                recordData: record
+            }
+        });
+    });
+
+    afterEach(() => {
+        wrapper = null;
+        router = null
+    });
+
+    it("displays breadcrumbs", () => {
+        expect(wrapper.findComponent({ name: 'breadCrumbs' }).exists()).toBe(true);
+    });
+
+    it("displays a work title", () => {
+        expect(wrapper.find('h2').text()).toEqual('Listen for real');
+    });
+
+    it("displays an iframe viewer for images", async () => {
+       let updated_record = cloneDeep(record);
+       updated_record.viewerType = 'uv';
+       await wrapper.setProps({
+           recordData: updated_record
+       });
+       expect(wrapper.find('iframe').exists()).toBe(true);
+    });
+
+    it("displays an iframe viewer for pdfs", async () => {
+        let updated_record = cloneDeep(record);
+        updated_record.viewerType = 'pdf';
+        await wrapper.setProps({
+            recordData: updated_record
+        });
+        expect(wrapper.find('iframe').exists()).toBe(true);
+    });
+
+    it("uses the audio player component for audio files", () => {
+        expect(wrapper.findComponent({ name: 'audioPlayer' }).exists()).toBe(true);
+    });
+
+    it("displays a list of files associated with the work", () => {
+        expect(wrapper.findComponent({ name: 'fileList' }).exists()).toBe(true);
+    });
+
+    it("displays a list of neighbor works", () => {
+        expect(wrapper.findComponent({ name: 'neighborList' }).exists()).toBe(true);
+    });
+
+    it("sets a link to its parent collection", () => {
+        expect(wrapper.find('.parent-collection').attributes('href')).toEqual(`record/${record.briefObject.parentCollectionId}`);
+    });
+
+    it("loads record MODS metadata as html", (done) => {
+        moxios.install();
+
+        const html_data = "<table><tr><th>Title</th><td><p>Listen for real</p></td></tr></table>";
+        const url = `record/${record.briefObject.id}/metadataView`;
+        moxios.stubRequest(new RegExp(url), {
+            status: 200,
+            response: html_data
+        });
+        wrapper.vm.loadMetadata();
+
+        moxios.wait(() => {
+            expect(wrapper.vm.metadata).toEqual(html_data);
+            expect(wrapper.find('#mods_data_display').text()).toEqual(expect.stringContaining('Listen for real'));
+            done();
+        });
+
+        moxios.uninstall();
+    });
```

Comment:
Might be good to test the behaviors of the download/request access buttons. Wouldn't hurt to add a test to verify that the finding aid field does and does not show depending on if a value is populated or not.